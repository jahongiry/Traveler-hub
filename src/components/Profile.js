import { useSelector } from 'react-redux';
import classes from './Profile.module.css';

const Profile = () => {
  const info = useSelector((state) => state.rocket);
  return (
    <div className={classes.mainContainer}>
      <div className={classes.missions}>
        <p>Under construction</p>
      </div>
      <div className={classes.rocket}>
        <h2>My Rockets</h2>
        <table>
          <tbody>
            {info.map((data) => (
              <tr key={data.id} className={classes.containerRocket}>
                {data.reserved && (
                  <th className={classes.items}>{data.rocket_name}</th>
                )}
              </tr>
            ))}
          </tbody>
        </table>
      </div>
    </div>
  );
};

export default Profile;
