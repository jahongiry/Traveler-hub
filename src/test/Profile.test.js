import { render } from '@testing-library/react';
// import { BrowserRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Profile from '../components/Profile';

describe('MissionDisplay', () => {
  test('Builds the snapchot of the Profile component safely', () => {
    const displayMission = renderer.create(<Profile />);
    expect(displayMission.toJSON()).toMatchSnapshot();
  });

  test('renders Profile component', () => {
    render(<Profile />);
  });
});
