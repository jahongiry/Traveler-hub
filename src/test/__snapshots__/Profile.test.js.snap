// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MissionDisplay Builds the snapchot of the Profile component safely 1`] = `null`;
